docs/articles/basics/basic_bot.md

```diff
@@ -7,23 +7,16 @@ In order for that to happen, you need to make a program that connects to discord
 This guide requires you to have solid basics in C#. Open your IDE. If you don't have one installed, I recommend installing 
 one of the following:
 
-* [Visual Studio 2017 Community](https://www.visualstudio.com/thank-you-downloading-visual-studio/?sku=Community&rel=15 "Visual Studio 2017 Community")
+* [Visual Studio 2019 Community](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=Community&rel=16)
 * [Visual Studio Code](https://code.visualstudio.com/download "Visual Studio Code")
 
-When installing Visual Studio 2017, make sure you select the **.NET Core cross-platform development** option.
+When installing Visual Studio 2019, make sure you select the **.NET Core cross-platform development** option.
```

Comment:
This is no longer applicable, .NET Core is installed with the .NET workload I believe. Will have to confirm this. At the very least, the specific workload is gone between 17 and 19.

Reply:
Will do.  I was gonna look at replacing all the images if they needed to be later this week so i will definitely hit that when i replace the images